src/Microsoft.DotNet.Interactive.Http.Tests/HttpKernelTests.cs

```diff
@@ -2311,6 +2311,73 @@ public async Task responses_to_named_requests_can_be_accessed_as_symbols_in_late
         secondResult.Events.Should().NotContainErrors();
     }
 
+    [Fact]
+    public async Task headers_for_named_requests_can_be_accessed_correctly()
```

Comment:
```suggestion
    public async Task Response_headers_for_named_requests_can_be_accessed_correctly()
```

Reply:
This name is a little more specific, since the test isn't checking request headers.